### PR TITLE
GTFS Part V : TransferRule and Frequency

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
@chicco785 please could you r?

File: UrbanMobility/Frequency/doc/spec.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  + Attribute type: Property. [Text](https://schema.org/Text)
+  + Optional
+  
++ `hasTrip`: Trip associated to this Entity.
+  + Attribute type: Relationship. It shall point to an Entity of Type [gtfs:Trip](../../Trip/spec.md)
+  + Mandatory
+
++ `startTme`: Same as GTFS `start_time`.
+    + Attribute type: Property. [Text](https://schema.org/Text).
+    + Mandatory
```
Comment (chicco785) [inline UrbanMobility/Frequency/doc/spec.md]: no discussion on the format?

File: UrbanMobility/Frequency/doc/spec.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    
++ `endTime`: Same as GTFS `end_time`.
+    + Attribute type: Property. [Text](https://schema.org/Text).
+    + Mandatory
```
Comment (chicco785) [inline UrbanMobility/Frequency/doc/spec.md]: see above

File: UrbanMobility/TransferRule/doc/spec.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  + Attribute type: Relationship. It shall point to an Entity of Type [gtfs:Stop](../../Stop/spec.md) or [gtfs:Station](../../Station/spec.md)
+  + Mandatory
+  
++ `transferType`: Same as GTFS `transfer_type`.
+  + Attribute type: Property. [Text](https://schema.org/Text).
+  + Allowed values: (`"0"`,`"1"`,`"2"`,`"3"`)
+  + Mandatory
+    
++ `minimumTransferTime`: Same as GTFS `min_transfer_time`.
+  + Attribute type: Property. [Integer](https://schema.org/Integer).
```
Comment (chicco785) [inline UrbanMobility/TransferRule/doc/spec.md]: what about the unit?

**Pull request merged**